docs/cli-spec.md

```diff
@@ -0,0 +1,47 @@
+# Signing CLI specficiation
+
+### Inputs
+
+The parameters to the signing client are as follows:
+
+`sign code AzureKeyVault [options] <files>`
+
+Options:
+
+- `-o | --output` - The output file. If omitted, overwrites input. Must be a directory if multiple files are specified.
+- `-b | --base-directory` - Base directory for files to override the working directory.
+- `-f | --force` - Overwrites a signature if it exists.
+- `-m | --max-concurrency` - Maximum concurrency (default is 4)
```

Comment:
Does this concurrency apply to connection to Azure Vault or timestamp server?

Reply:
it's in areas of the code that process multiple files at the same time -- i.e., looping over all files, reading, signing, calling api's. It's for a `Parallel.ForEach` effectively.